src/mjolnir/pbfgraphparser.cc

```diff
@@ -589,6 +589,19 @@ struct graph_callback : public OSMPBF::Callback {
       osmdata_.access_restrictions.insert(
           AccessRestrictionsMultiMap::value_type(osmid_, restriction));
     };
+    tag_handlers_["hov_type"] = [this]() {
+      // If this tag is set then the way is either HOV-2 or HOV-3.
+      // There are no other real-world hov levels.
+      std::string hov_type = tag_.second;
+      if (hov_type == "HOV2") {
+        way_.set_hov_type(valhalla::baldr::HOVEdgeType::kHOV2);
+      } else if (hov_type == "HOV3") {
+        way_.set_hov_type(valhalla::baldr::HOVEdgeType::kHOV3);
+      } else {
+        // assert? For now just set to the max hov level.
+        way_.set_hov_type(valhalla::baldr::HOVEdgeType::kHOV3);
```

Comment:
maybe warn and ignore is better?

Reply:
Good call on `LOG_WARN`.  I still think we should set it to the max hov level... to ignore would leave it at `kHOV2`.